### DISCUSSION TITLE
How to override files in /libraries/astroid/

### DISCUSSION THREAD

**Discussion by vtheod** [templaza/astroid-framework#255] [Q&A] [answered]
Hey all!

I want to ask if there is a way to override some files in /libraries/astroid/framework/frontend/blog/modules

Thanks in advance,
Vassilis

**Comment by sonvnn** [comment, marked as answer]
Hi @vtheod 

You can use `/templates/{your-template}/html/frontend/blog/modules` to override library.

Please let me know if you need further assistant. 

Best Regards,
Sonny

**Comment by vtheod** [comment]
Many thanks @sonvnn!!

**Comment by Chacapamac** [comment]
Nice to know!
Good Day Everybody!
